ansible/Dockerfile

```diff
@@ -1,5 +1,16 @@
 FROM alpine:3.6
-RUN apk update
-RUN apk add git \
-    ansible
-RUN echo "'Done !"
\ No newline at end of file
+
+ARG CI_HELPER_VERSION
+ARG GLIBC_VERSION
+ARG PYTHON_VERSION
+
+WORKDIR /root
+RUN apk --update --no-cache add bash ca-certificates curl git openssh-client python=${PYTHON_VERSION} py-pip ansible=${VERSION} && \
```

Comment:
For easier debugging when the build of the Docker image fails, you should surround the `apk` command with `echo` commands, like it's done for CI Helper:
```docker
RUN echo "Install Ansible" && \
    apk --update --no-cache add bash ca-certificates curl git openssh-client python=${PYTHON_VERSION} py-pip ansible=${VERSION} && \
    echo "Done Install Ansible" && \
…
```
It helps pinpointing the failure, since the `RUN` command is actually composed of a long chain of commands. It's also consistent with the other `Dockerfile`s.